src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,54 @@
+@import "../../style/variables.scss";
+@import "../../style/mixin.scss";
+
+.introBox {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.introduce {
+  margin-bottom: 20px;
+  padding: 20px;
+  width: 200px;
+  height: 300px;
+  border: 1px solid $color-black;
+  border-radius: 10px;
+
+  &> img {
+  margin:0 0 10px 0;
```

```suggestion
  margin-bottom:10px;
```
이렇게 스타일을 주는 것이 더 명확할 것 같습니다